products/ceqr_survey/models/intermediate/int__all_buffers.sql

```diff
@@ -0,0 +1,18 @@
+WITH all_buffers AS (  -- noqa: disable=all
+{{ dbt_utils.union_relations(
+    relations=[
+        ref('int__elevated_railways'),
+        ref('int__nysdec_state_facility_permits'),
+        ref('int__nysdec_title_v_facility_permits'),
+        ref('int__dep_cats_permits'),
+        ref('stg__dcm_arterial_highways'),
+        ref('stg__dcp_air_quality_vent_towers_buffered'),
+        ref('stg__panynj_airports')
+    ],
+    column_override={"raw_geom": "geometry", "buffer": "geometry"}
+) }}
+)
+-- Note: without `column_override`, dbt throws an error trying to cast.
+-- e.g.: `cast("raw_geom" as USER-DEFINED) as "raw...`
+
+SELECT * FROM all_buffers
```

Comment:
nit: because of `select *` we end up with a `_dbt_source_relation` column. might be useful for debugging, but it's unexpected based on model's config in the yml

Reply:
@damonmcc Good to know!